src/components/CryptoCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./CryptoCard.css";
+
+type CryptoCardProps = {
+  coinId: string;
+  coinName: string;
+  usd: number;
+  eur: number;
+  gbp: number;
```

Comment:
prefer a single prop `prices` whith an array of currency and value. That way you could keep this dynamic

Reply:
These three were the only currencies required as per the assignment, but I have made this dynamic in the latest commit